torch/fx/experimental/proxy_tensor.py

```diff
@@ -232,6 +231,18 @@ def maybe_disable_fake_tensor_mode():
     # library
     return unset_fake_temporarily()
 
+def _unset_proxy_mode(pre_dispatch: bool = False):
+    if pre_dispatch:
+        mode = unset_mode_pre_dispatch(torch._C._TorchDispatchModeKey.PROXY)
+        return mode
+    return torch._C._unset_dispatch_mode(torch._C._TorchDispatchModeKey.PROXY)
+
+def _set_proxy_mode(mode, pre_dispatch: bool = False):
```

Comment:
This also shouldn't need the kwarg, since the `pre_dispatch` DispatchKey should already be directly on the mode